src/geom/SpherePoint.cc

```diff
@@ -134,12 +138,6 @@ bool SpherePoint::operator==(SpherePoint const& other) const noexcept {
 bool SpherePoint::operator!=(SpherePoint const& other) const noexcept { return !(*this == other); }
 
 Angle SpherePoint::bearingTo(SpherePoint const& other) const {
-    if (atPole()) {
-        stringstream buffer;
-        buffer << "Cannot calculate offset from pole " << *this << ".";
-        throw pexExcept::DomainError(buffer.str());
-    }
-
```

Comment:
As I understand it, `bearingTo` can give different results for the same point (e.g., the north pole) depending on how the longitude has been initialized. If this is the desired behavior, consider adding a warning to the documentation to make this explicit.

Reply:
I think the documentation is sufficiently clear, now that I have reworked it.